Dockerfile

```diff
@@ -1 +1,78 @@
+# set the base image tag with ARG
+ARG BASE_IMAGE=ubuntu
+FROM $BASE_IMAGE
```

Comment:
Please can you specify the tag used? The tag should should also be a user input.